makefile

```diff
@@ -84,4 +84,14 @@ doc-preview: doc-build
 	@(cd doc && npm run preview)
 
 doc-deploy:
-	@(cd doc && npx gh-pages -d dist -e doc -b gh-pages -m "Deploy docs" --remove doc)
+	@(cd doc && npx gh-pages -b gh-pages -d dist -e doc -m "Deploy docs" --remove doc)
+
+registry-build: ## build the shadcn registry
```

Comment:
explain in a comment that it's called in a CI pipeline, it shouldn't be called by hand